ui/src/app/ui-banner/ui-banner.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from 'react';
+require('./ui-banner.scss');
+
+export interface UiBannerProps extends React.Props<any> {
+    items: {apiUrl:string, fields: Array<string> };
```

Comment:
To be consistent with the rest of UI code we should add a "service" that returns API response with typed output and implement rendering in the banner component. We can make it generic later.

Reply:
This issue has been addressed.